cmd/root.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+    "fmt"
+    "os"
+    "github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+    Use:   "ampyq",
+    Short: "Amrita PYQ CLI",
+    Long:  `A CLI application to access Amrita Repository for previous year question papers.`,
+    Run: func(cmd *cobra.Command, args []string) {
+		fmt.Print(logo_ascii)
+        start()
+    },
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+func Execute() {
+    if err := rootCmd.Execute(); err != nil {
+        fmt.Println(err)
+        os.Exit(1)
+    }
+}
+
+// start function - equivalent to start() in Python
+func start() {
+    fmt.Println("Available Courses:\n")
```

Comment:
while `Println` already adds a `\n` can we use `backtik` for strings?

Reply:
While fmt.Println automatically adds a newline character (\n), using backticks in Go strings is useful for raw string literals. Raw string literals are enclosed in backticks (`) and can include line breaks and other unescaped characters.
Eg:-
fmt.Println(`Available Courses:
- Course 1
- Course 2
- Course 3`)
In this case, backticks allow you to include the list directly within the string without needing escape characters.
For existing code with no special characters or line breaks, using double quotes is just as effective.
But I have made the changes in the latest version as this has been mentioned here.